dispatcher.go

```diff
@@ -225,8 +269,8 @@ func (b *backendNF) connectToServer() {
 			req.Msgtype = gClient.MsgType_INIT_MSG
 			req.SctplbId = os.Getenv("HOSTNAME")
 			candidate := value.(*context.Ran)
-			if candidate.RanId != nil {
-				req.GnbId = *candidate.RanId
+			if candidate.RanId != "" {
```

Comment:
Why this? Given that RanId is a string and its default value is empty (""), should not the check be as follows:

```suggestion
			if candidate != nil {
```